tsv_file_parser.conf

```diff
@@ -0,0 +1,18 @@
+input{
+    file{
+        path => 'FILEPATH'
+		type => 'INDEX_TYPE'
+        start_position => 'beginning'
+        sincedb_path => '/dev/null'
+    }}
+filter {
+	grok {
+    match => { 'message' => '(%{DATA:ID}	%{BASE16FLOAT:AFR}	%{BASE16FLOAT:AMR}	%{BASE16FLOAT:EAS}	%{BASE16FLOAT:EUR}	%{BASE16FLOAT:SAS}	)' }
+  }}
+
+output{ 
+  elasticsearch {
+    codec => json
+    hosts => ['ES_COORDINATOR','ES_MASTER']  
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/aashishchauhan06/logstash-config/pull/7#issuecomment-2155163474>#49d7c2</a><span> - 06/07/2024, 04:30 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Hardcoded Elasticsearch host names ('ES_COORDINATOR', 'ES_MASTER') can lead to configuration errors in different environments and hinder scalability and flexibility.
</div>
<div id="fix">
<b>Fix:</b> Use environment variables or external configuration settings for Elasticsearch host names to allow flexibility and environment-specific configurations.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div>
<pre><code>
 @@ -16,1 +16,1 @@
 -    hosts => ['ES_COORDINATOR','ES_MASTER']
 +    hosts => ['${ES_HOSTS}']
</code></pre>
</div>
</details>
</div>
<br>


</div>